ui/i18n/messages.json

```
@@ -81,6 +81,16 @@
   "documentAclTable.until": "Until",
   "documentAclTable.untilMiddle": "until",
   "documentAclTable.userGroup": "User / Group",
+  "documentContent.moveToColdStorage.ask.confirmation":"Send to cold storage?",
```

Seems rather abrupt. Perhaps something like
```suggestion
  "documentContent.moveToColdStorage.ask.confirmation":"Do you want to send the document content to cold storage?",
```

---

for this message I respected what Carolina asked in her mockup :). but perhaps we can change it @carolinagouveia

---

I'm was just thinking that we should make it clear to the user what's being moved to cold storage (the document content).